File: max_profit.js
var maxProfit = function (prices) {
  let profits = 0;
  let diff = 0;

  for (let i = 1; i < prices.length; i++) {
    diff = prices[i] - prices[i - 1];
    if (diff > 0) {
      profits += diff;
    }
  }

  return profits;
};

File: max_profit.py
class Solution:
    def maxProfit(self, prices: List[int]) -> int:
        profit = 0
        ranges = range(len(prices) - 1)

        for i in ranges:
            diff = prices[i+1] - prices[i]
            if diff > 0:
                profit += diff

        return profit
